lib/events/dynamoevents/dynamoevents.go

```diff
@@ -126,6 +123,19 @@ type Config struct {
 	WriteMinCapacity int64
 	// WriteTargetValue is the ratio of consumed write to provisioned capacity.
 	WriteTargetValue float64
+
+	// UseFIPSEndpoint uses AWS FedRAMP/FIPS 140-2 mode endpoints. This is a pointer because AWS uses three states
+	// to determine its behavior:
+	// Unset - allows environment variables or AWS config to set the value
+	// Enabled - explicitly enable
+	// Disabled - explicitly disable
+	UseFIPSEndpoint *bool
```

Comment:
We probably want the default behavior after this change to match the behavior we have today. Does that mean we should default it to false instead of nil?

Reply:
Right now the behavior is `Unset`, as a current workaround we can use `AWS_USE_FIPS_ENDPOINT=true` to enable `FIPS` endpoints for AWS services, so I think `nil` is the appropriate value.